src/pages/office/components/additional-information/additional-information.ts

```diff
@@ -17,11 +22,12 @@ export class AdditionalInformationComponent implements OnChanges {
   additionalInformationChange = new EventEmitter<string>();
 
   private formControl: FormControl;
+  private fieldName: string = 'additionalInformation';
```

Comment:
this can turned into a constant by using the `static readonly` configuration

Reply:
Nice. will do this for all components